.bc-exclude.php

```diff
@@ -34,6 +31,7 @@
         'ADDED: Parameter label was added to Method __construct\(\) of class Shopware\\\\Core\\\\Checkout\\\\Cart\\\\Tax\\\\Struct\\\\CalculatedTax',
         'ADDED: Parameter senderName was added to Method __construct\(\) of class Shopware\\\\Core\\\\Content\\\\Mail\\\\Service\\\\SendMailTemplateParams',
         'ADDED: Parameter response was added to Method __construct\(\) of class Shopware\\\\Elasticsearch\\\\Framework\\\\DataAbstractionLayer\\\\Event\\\\ElasticsearchEntitySearcherSearchedEvent',
+        'ADDED: Parameter clock was added to Method __construct\(\) of class Shopware\\\\Core\\\\Checkout\\\\Promotion\\\\Gateway\\\\Template\\\\ActiveDateRange',
```

Comment:
Because I see a lot of string here that do not use regex: Maybe we should use `preg_quote` for those instead of escaping manually, makes it much easier to read

Reply:
yes, wrapping each line in it: `\preg_quote('ADDED: Parameter clock was added to Method __construct() ...'),`